test/compliment/sources/t_local_bindings.clj

```diff
@@ -82,10 +82,13 @@
                                          '(let [foo 42,
                                                 [bar baz] lst
                                                 {a :a, {b :b :as c} :b, [d] :d} m
-                                                {:keys [key1 key2] :strs [key3]} m2
+                                                {:keys [key1 key2] :strs [key3] :syms [key4]} m2
+                                                {:keys [:a/key5 :b/key6]} m3
```

Comment:
I think there should also be coverage for `{:keys [::a/keyN ::b/keyM]}` (i.e. ns-aliased)